visualization/app/codeCharta/core/multipleFile/multipleFile.service.ts

```diff
@@ -93,16 +93,16 @@ export class MultipleFileService {
         let outputNode: CodeMapNode = {} as CodeMapNode;
         outputNode["name"]= inputCodeMap.fileName;
 
-        for(let property in inputCodeMap.root){
+        for(let property in inputCodeMap.nodes){
 
             if(property == "children"){
-                outputNode[property] = JSON.parse(JSON.stringify(inputCodeMap.root.children));
+                outputNode[property] = JSON.parse(JSON.stringify(inputCodeMap.nodes.children));
```

Comment:
check if needed

Reply:
This `multipleFile.service` is completely refactored in PR #314

Reply:
btw: just removed the `stringily` methods on the corresponding branch for #314